gateway/grpc/README.md

```diff
@@ -16,26 +16,36 @@ gcloud container  clusters create cluster-1 --machine-type "n1-standard-2" \
  --cluster-version "1.20"  -q
 ```
 
+Install Gateway CRDs
+
+```
+kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.3.0" \
+| kubectl apply -f -
+```
 
-Create SSL Certificate
 
-Note, as of `5/6/21`, Gateway does not support [spec.listeners.tls.certificateRefs](https://cloud.google.com/kubernetes-engine/docs/how-to/gatewayclass-capabilities#gateway) so we have to upload a static cert to GCP.  Note we are creating two sets of certificates scoped for global and regional to account for external LB (global) and ILB (regional)
+optionally create SSL Certificate for use with statically defined certificates (`networking.gke.io/pre-shared-certs`)
```

Comment:
Can you please add a link here to the doc for Google managed certs https://cloud.google.com/kubernetes-engine/docs/how-to/managed-certs

And also change from Statically defined certificates to Google Managed Certificates. we would like to keep this in line with the doc

Reply:
its easier for a demo to NOT use managed certs as it requires a whole lot of other things wlike DNS,etc

for a sample here, even static certs work so i'll reword it use uploaded certs and then optionally google -managed